generate-java.sh

```diff
@@ -2,6 +2,7 @@
 
 dir_resolve()
 {
+"$1"!="" && mkdir -p "$1"
```

Comment:
The target is not mandatory for the go generator, so actually I was aligning the two with this change.